<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QStackedWidget" name="pagesStack">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>450</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>320</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>320</width>
     <height>480</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Helvetica Neue</family>
    </font>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="WorkoutsPage">
    <property name="minimumSize">
     <size>
      <width>320</width>
      <height>450</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>320</width>
      <height>450</height>
     </size>
    </property>
    <widget class="QStackedWidget" name="workoutsStack">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>321</width>
       <height>451</height>
      </rect>
     </property>
     <widget class="QWidget" name="workoutsMainPage">
      <widget class="QPushButton" name="addWorkoutButton">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>40</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
      <widget class="QPushButton" name="editWorkoutButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
      <widget class="QListWidget" name="workoutList">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>281</width>
         <height>291</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="workoutsTitle">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>10</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Helvetica Neue</family>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="text">
        <string>WORKOUTS</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="startWorkoutButton">
       <property name="geometry">
        <rect>
         <x>14</x>
         <y>380</y>
         <width>293</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Start Workout</string>
       </property>
      </widget>
      <widget class="QPushButton" name="deleteWorkoutButton">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>40</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="addWorkoutPage">
      <widget class="QLabel" name="addWorkoutTitle">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>10</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Helvetica Neue</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ADD WORKOUT</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="addWorkoutNameLabel">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Name:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="addWorkoutNameDoneButton">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>180</y>
         <width>261</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>DONE</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="addWorkoutNameLine">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>130</y>
         <width>261</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="editWorkoutPage">
      <widget class="QLineEdit" name="editWorkoutNameLine">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>281</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="editWorkoutNameDoneButton">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>410</y>
         <width>261</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>DONE</string>
       </property>
      </widget>
      <widget class="QLabel" name="editWorkoutExercisesLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Exercises:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="editWorkoutTitle">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>10</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Helvetica Neue</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>EDIT WORKOUT</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="editWorkoutNameLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Name:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QListWidget" name="editWorkoutExercisesList">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>281</width>
         <height>261</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="deleteWorkoutButton_3">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>110</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
      <widget class="QPushButton" name="addWorkoutButton_3">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>110</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="performWorkoutPage"/>
    </widget>
   </widget>
   <widget class="QWidget" name="ExercisesPage">
    <property name="minimumSize">
     <size>
      <width>320</width>
      <height>450</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>320</width>
      <height>450</height>
     </size>
    </property>
    <widget class="QStackedWidget" name="exercisesStack">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>321</width>
       <height>451</height>
      </rect>
     </property>
     <widget class="QWidget" name="exercisesMainPage">
      <widget class="QPushButton" name="deleteExerciseButton">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>40</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
      <widget class="QLabel" name="exercisesTitle">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>10</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Helvetica Neue</family>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="text">
        <string>EXERCISES</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="addExerciseButton">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>40</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
      <widget class="QListWidget" name="exerciseList">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>281</width>
         <height>331</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="editExerciseButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="addExercisePage">
      <widget class="QLabel" name="addExerciseTitle">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>10</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Helvetica Neue</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ADD EXERCISE</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="addExerciseNameLabel">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Name:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="addExerciseNameDoneButton">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>180</y>
         <width>261</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>DONE</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="addExerciseNameLine">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>130</y>
         <width>261</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="HistoryPage">
    <property name="minimumSize">
     <size>
      <width>320</width>
      <height>450</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>320</width>
      <height>450</height>
     </size>
    </property>
    <widget class="QLabel" name="historyTitle">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica Neue</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text">
      <string>HISTORY</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="StatsPage">
    <property name="minimumSize">
     <size>
      <width>320</width>
      <height>450</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>320</width>
      <height>450</height>
     </size>
    </property>
    <widget class="QLabel" name="statsTitle">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica Neue</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text">
      <string>STATISTICS</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="workoutsButton">
   <property name="geometry">
    <rect>
     <x>-6</x>
     <y>450</y>
     <width>93</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Workouts</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exercisesButton">
   <property name="geometry">
    <rect>
     <x>74</x>
     <y>450</y>
     <width>93</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Exercises</string>
   </property>
  </widget>
  <widget class="QPushButton" name="historyButton">
   <property name="geometry">
    <rect>
     <x>154</x>
     <y>450</y>
     <width>93</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>History</string>
   </property>
  </widget>
  <widget class="QPushButton" name="statsButton">
   <property name="geometry">
    <rect>
     <x>234</x>
     <y>450</y>
     <width>93</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Stats</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
